.github/workflows/test.yml

```
@@ -88,6 +88,13 @@ jobs:
           npm run eql:download
           npm run eql:install
         working-directory: ./integration-tests
+      - name: Create .env file in ./integration-tests/
+        run: |
+          touch ./integration-tests/.env
```

Not needed, because the `>>` in the next steps will create it:

```suggestion
          touch ./integration-tests/.env
```